WalletWasabi/Helpers/EnvironmentHelpers.cs

```diff
@@ -195,5 +195,20 @@ public static string GetMethodName([CallerMemberName] string callerName = "")
 		{
 			return callerName;
 		}
+
+		public static string GetFullBaseDirectory()
+		{
+			var fullBaseDirectory = Path.GetFullPath(AppContext.BaseDirectory);
+
+			if (!RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
+			{
+				if (!fullBaseDirectory.StartsWith('/'))
+				{
+					fullBaseDirectory.Insert(0, "/");
```

Comment:
This cannot work because strings are immutable.

```suggestion
					fullBaseDirectory = fullBaseDirectory.Insert(0,** "/");
```

Reply:
This could be the issue with https://github.com/zkSNACKs/WalletWasabi/issues/2289 ?